### ISSUE TITLE
Add Auto Attack

### ISSUE THREAD

**Issue by umarkhan852** [epic-hacks/epic-hacks-surviv.io#10]
Pls add auto attack like ice hack and add auto aim
pls

**Comment by fire-and-ice582** [comment]
Could you add just the aimbot for the free version and all the other bonuses in the premium?

**Comment by epic-hacks** [comment]
@umarkhan852 we are __not__ planning to add autoattack and autoaim in the nearest time. Because that way it would be obvious to other players that you are cheating. And we want the game to look natural even with cheats

**Comment by epic-hacks** [comment]
@fire-and-ice582 probably not

**Comment by humphreygaming** [comment]
@umarkhan852 lemme rephrase that. Adding auto-attack would make it look way to much like SCI, which was already skidded by this kid

**Comment by epic-hacks** [comment]
@humphreygaming true, it would look like SCI. And that's what we don't want to happen 😉

**Comment by humphreygaming** [comment]
@epic-hacks Does that also explain the extremely similar background.js file structure? Does that also explain how it has the same menu appearance and a coincidental "pan hero"? Does that also explain the names of the cheat chunks and the base64 encoded image?

**Comment by epic-hacks** [comment]
@humphreygaming unpack-webpack + deobfuscator + some time = pretty much source code

**Comment by humphreygaming** [comment]
@epic-hacks its against our terms and licence. Want a DMCA takedown?

**Comment by epic-hacks** [comment]
@humphreygaming  who said that we are using your cheat in ours? we just looked at your source, which is not prohibited. That's it.

**Comment by humphreygaming** [comment]
you literally copy pasted all the "from" and "to" things to your own. How is that only looking at the source

**Comment by epic-hacks** [comment]
nope, we have taken a look at your approach in order to create our version.
How is that even possible to debundle your code in the way that it would be working right out of the box? Maybe there is a way, but i didn't find it.
Therefore, we've only looked at your structure. by the way it's good enough.
Moreover, if you will use the same approach and debundle our code, you'll see that all the cheats are written from scratch 🤷

Let me justify. We were not going against your license. So, just take it as it is. You develop your cheat, we develop ours. And maybe we will even collab in the future
